desserts/mango-coconut-rice.md

```diff
@@ -0,0 +1,31 @@
+# Mango coconut rice
+
+This Thai dessert compliments salty and spicy food well. This recipe is inspired by a Food Network [mango coconut rice recipe](https://www.foodnetwork.com/recipes/mango-coconut-rice-recipe-1937791) from Paula Dean and a [caramelized mango recipe](https://www.myrecipes.com/recipe/caramelized-mangoes) from MyRecipes.
+
+
+## Ingredients
+
+- 1 tablespoon olive oil
```

Comment:
Should these measurements use abbreviations? It is how we have other recipes here.